README.md

```diff
@@ -197,6 +200,46 @@ onConsentChange(({ tcfv2, ccpa }) => {
 });
 ```
 
+## Get consent for particular vendor
```

Comment:
maybe this section can be collapsed into `Using Consent`? so it discusses both `getConsentFor` and `onConsentChange`?